IdentityCore/src/util/NSJSONSerialization+MSIDExtensions.m

```diff
@@ -28,6 +28,11 @@ @implementation NSJSONSerialization (MSIDExtensions)
 
 + (NSDictionary *)msidNormalizedDictionaryFromJsonData:(NSData *)data error:(NSError **)error
 {
+    if (!data)
```

Comment:
nit: is it good practice to check both !data and [data length] == 0, just to be safe? I was updated in my PR  yesterday following similar Rohit's comment about that.

Reply:
For my understanding, will ```[data length] == 0``` not always give 0 if data is nil?

Reply:
Cool, thanks Kai for the tip!

Reply:
Addressed.